src/promptflow/promptflow/_sdk/_load_functions.py

```diff
@@ -72,7 +73,7 @@ def load_flow(
     :return: A Flow object
     :rtype: Flow
     """
-    return ProtectedFlow.load(source)
+    return Flow.load(source, **kwargs)
```

Comment:
after some other practice I agree on this change, but wonder if this is safe as we already have some logic depending on ProtectedFlow